src/transformers/modeling_tf_bert.py

```diff
@@ -212,62 +258,63 @@ def __init__(self, config, **kwargs):
             )
 
         self.num_attention_heads = config.num_attention_heads
-        assert config.hidden_size % config.num_attention_heads == 0
```

Comment:
why delete the assert? Shouldn't we instead add an error message here?

Reply:
There is already a error message, take a look to the lines above :)

```
if config.hidden_size % config.num_attention_heads != 0:
            raise ValueError(
                "The hidden size (%d) is not a multiple of the number of attention "
                "heads (%d)" % (config.hidden_size, config.num_attention_heads)
            )
```